src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerEngine.java

```diff
@@ -767,8 +846,12 @@ public List<VariantContext> callRegion(final AssemblyRegion region, final Featur
 
         //Realign reads to their best haplotype.
         final SWParameters readToHaplotypeSWParameters = hcArgs.getReadToHaplotypeSWParameters();
-        final Map<GATKRead, GATKRead> readRealignments = AssemblyBasedCallerUtils.realignReadsToTheirBestHaplotype(subsettedReadLikelihoodsFinal, assemblyResult.getReferenceHaplotype(), assemblyResult.getPaddedReferenceLoc(), aligner, readToHaplotypeSWParameters);
-        subsettedReadLikelihoodsFinal.changeEvidence(readRealignments);
+        // TODO Yes we skip realignment entirely when we are in DRAGEN-GATK mode. Realignment of the reads makes no sense when
```

Comment:
DRAGEN doesn't remap the reads at all internally to my knowledge. In the first DRAGEN-GATK pass I used to have an argument `--use-original-alignments-for-genotyping-overlap` to deal with this... I got rid of it in the review at the time because it didn't overwhelmingly affect FE.
The problem is that now PDHMM is incompatable....